<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>807</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>807</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SR Weight Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lin_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Taper 1</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="com_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="com_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lin_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Taper 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lin_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Taper 3</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lin_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Taper 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STRING DETAILS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lin_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Taper 5</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="com_7">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="currentText">
      <string>select rod length</string>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>320</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_8">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="com_9">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="com_10">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="com_11">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>91</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Number of rods</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rod length</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>120</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total string weight</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lin_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Taper 6</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>select rod length</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>----</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/8 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/4 inch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 1/2 inch</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="lb_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Klbs</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="ton_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tons</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calc_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>CALCULATE</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/PROGRAMMING/Images/Icons/calculating.png</normaloff>D:/PROGRAMMING/Images/Icons/calculating.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total rod string weight</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OUTPUT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="buy_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consider Bouyancy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="api_btn">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>120</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Fluid  API</string>
     </property>
    </widget>
    <widget class="QLabel" name="unit_lbl">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Klbs</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>681</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:400; color:#9e0000;&quot;&gt;SUCKER ROD STRING WEIGHT CALCULATOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
